<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconCfgBase</class>
 <widget class="QWidget" name="IconCfgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnUp">
     <property name="text">
      <string>&amp;Up</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnDown">
     <property name="text">
      <string>&amp;Down</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnRemove">
     <property name="text">
      <string>&amp;Remove</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>&amp;New</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="Spacer5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QListWidget" name="lstIcon"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnUp</tabstop>
  <tabstop>btnDown</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>lstIcon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
